static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -1,12 +1,9 @@
-import { mocked } from 'ts-jest/utils';
-import raven_ from 'lib/raven';
+import raven from 'lib/raven';
 import config_ from '../../lib/config';
 import { amIUsed } from './sentinel';
 
-const raven = raven_;
-
 const config = config_ as {
-	get: (s: string, d: boolean) => boolean;
+	get: jest.MockedFunction<(s: string, d: boolean) => boolean>;
 };
 
 jest.mock('lib/raven', () => ({
```

Comment:
I think you can get rid of this mock, too!

Reply:
I tried to use Jest's auto mocking but it wouldn't play ball for some reason 🤔